### DISCUSSION TITLE
Questions on KS ROMs and RP9

### DISCUSSION THREAD

**Discussion by SmokinDeist** [BlitterStudio/amiberry#765] [Q&A] [answered]
I have Amiga Forever 8 and a full set of Kickstart ROMs to play with. With some of the other suggested ROMs not absolutely required, do you have suggested or expected names for them?

AF allows me to create .RP9 files and I have some stuff I like to play in that format. There is pretty much no mention of support for this other than at Recalbox's site. Is this actually supported? If so, do I just drop it in the ROMs folder with the rest of the games or is there anything else I would need to do?

**Comment by midwan** [comment, marked as answer]
Hi there!
The actual kickstart filenames don't matter, as they are checked and matched with their checksum values. You can keep the original filenames and just copy them in the kickstarts directory, then do a Rescan to have Amiberry pick them up.
Regarding rp9, i removed support for it in the latest beta, as nobody ever used it in all the years since I started this project. You're the first to ask about it! :)

**Reply by SmokinDeist** [reply]
Well, I'll have to hunt for a few ADFs then. I did try plopping a couple RP9s in and they did work seamlessly. 

I am using this on a Raspberry Pi 4 B 8GB under Recalbox.

**Reply by midwan** [reply]
Perhaps the best solution would the pre-installed WHDLoad packs, instead. Amiberry includes a "whdbooter" feature, which allows you to boot those .lha files directly, and also picks the best known emulator settings per game for you.

**Reply by kiasanth** [reply]
Unfortunately not all games have been converted to WHDLoad and only a small subset even work on it.

I personally have converted a lot of games to RP9 and would love this feature on my Pi400.
